recipes/qxlsx/all/conanfile.py

```diff
@@ -23,14 +25,7 @@ class QXlsxConan(ConanFile):
         "fPIC": True
     }
 
-    generators = "cmake", "cmake_find_package_multi"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def export_sources(self):
-        self.copy("CMakeLists.txt")
         for patch in self.conan_data.get("patches", {}).get(self.version, []):
             self.copy(patch["patch_file"])
```

Comment:
```suggestion
        for p in self.conan_data.get("patches", {}).get(self.version, []):
            copy(self, p["patch_file"], self.recipe_folder, self.export_sources_folder)
```
Prefer conan.tools.files.copy